provider/marathon/marathon_test.go

```diff
@@ -99,6 +99,7 @@ func TestMarathonLoadConfig(t *testing.T) {
 				`frontend-test`: {
 					Backend:        "backend-test",
 					PassHostHeader: true,
+					BasicAuth:      []string{},
```

Comment:
Do the unaffected tests fail if we do not add empty `BasicAuth` slices?

We should not extend them unless it's required.